code/src/components/InputForm.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+const InputForm = ({input, setInput}) => {
+    
+    return (
+        <div className="input-form-div">
+            <label className="input-fields">
+            Your name is:  {input.name}
+            <input
+            type="text"
+            name="name"
+            onChange={event => setInput({...input, [event.target.name]: event.target.value})}
+            value={input.name}
+            required
+            />
+            </label>
```

Comment:
Do you remember about htmlFor and id attribute? That you can use to link your label to your input. This helps a lot with accessibility.